sql/hive/compatibility/src/test/scala/org/apache/spark/sql/hive/execution/HiveCompatibilitySuite.scala

```diff
@@ -265,6 +265,32 @@ class HiveCompatibilitySuite extends HiveQueryFileTest with BeforeAndAfter {
     // the answer is sensitive for jdk version
     "udf_java_method",
 
+    // TODO Hive window function cannot be serialized in generating the golden file
+    "subquery_in",
+    // As we don't support the outer UDAF function used in the correlated query, combined with
+    // outer having clause: e.g.:
+    //    select b.key, min(b.value)
+    //    from src b
+    //    group by b.key
+    //    having exists ( select a.key
+    //    from src a
+    //      where a.value > 'val_9' and a.value = min(b.value)
+    //    )
+    // It throws exception like
+    // "cannot resolve 'b.value' given input columns key, _c1, key, value;"
+    // As the outer aggregation doesn't output the field 'value, we need rule
+    // for further resovling the having expressions.
+    "subquery_notin_having",
+    "subquery_exists_having",
```

Comment:
I mean the case the referenced attribute in the aggregation function is not explicit provided in the outer query case.